src/Integration.Vsix.UnitTests/SonarLintTracker/IssuesSnapshotTests.cs

```diff
@@ -140,6 +139,10 @@ public void HelpLink()
         {
             issue.RuleKey = "javascript:123";
             GetValue(StandardTableKeyNames.HelpLink).Should().Be("https://rules.sonarsource.com/javascript/RSPEC-123");
+            issue.RuleKey = "javascript:S123";
```

Comment:
Where do the keys come from? Why would it sometimes be with S and sometimes without?

Reply:
It should always be "S" but as it seems to have change I prefer to keep the old test.